### PR TITLE
Propagate analytics events in access: events and NO data

### PR DESCRIPTION
**Pull request by dvoytenko**
/cc @avimehta

### CODE REVIEW

File: extensions/amp-access/0.1/amp-access.js [hunk 1]
```diff
@@ -161,6 +169,13 @@ export class AccessService {
 
     /** @private {?Promise} */
     this.loginPromise_ = null;
+
+    /** @private {!Promise<InstrumentationService>} */
+    this.analyticsPromise_ = analyticsFor(this.win);
```
Comment (cramforce) [inline extensions/amp-access/0.1/amp-access.js]: Its mandatory to exist, right?

**Comment by cramforce** [comment]
This looks great.

Regarding docs. Maybe this should live with access docs?

The docs here https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#triggers need a rework I think @rudygalfi. The available events should probably get their own heading.

**Comment by rudygalfi** [comment]
@cramforce Revised doc: #1897

File: extensions/amp-access/0.1/amp-access.js (hunk 1 above)
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-access.js]: Correct.

**Comment by dvoytenko** [comment]
@cramforce @rudygalfi Alright. I will add docs in the access spec and make references from analytics docs there.

**Comment by rudygalfi** [comment]
Sorry, forgot to give my take on the docs question.

TL;DR: Agree with putting this in the access docs but may have further feedback on location once there's a draft to review.

I definitely think we should consolidate the key points regarding analytics for access in the access docs. That way it's discoverable by the people who are using the access solution. I think this would cover topics like the new trigger types and AUTHDATA variable. There may be other things.

I would also recommend incorporating documentation into the amp-analytics.md and substitutions files for completeness. It doesn't need to cover these things in depth. We could link off to the access docs to do that. It's a bit hard to suggest exactly how this should be handled, so I'd recommend preparing docs mainly in the access docs as you suggest @dvoytenko and we can discuss further in the review stages.

**Comment by dvoytenko** [comment]
@cramforce @rudygalfi I added `amp-access-analytics.md` docs file and linked it from the access and analytics specs. PTAL.

One question: I'm wondering if we should get rid of the `amp-` prefix in all the events.

**Comment by cramforce** [comment]
LGTM
